code/XMLTester.xml

```diff
@@ -440,6 +440,24 @@
 </participant>
 <!--><-->
 
+<!-->CONF-SHM-TRANSPORT-SETTING<-->
```

Comment:
In general, I think it help to see the same comments in the XML and the C++. I'll add those here

Reply:
What comments are you refering to? This descriptor is very simple, it only has a field <type>SHM</type>

Reply:
I mena the comments on the snippet `CONF-SHM-TRANSPORT-SETTING` in CodeTester.cpp